### PR TITLE
added New JS question

### PR DESCRIPTION
**Pull request by Dhanu084**

### CODE REVIEW

File: javascript/index.html [hunk 1]
```diff
@@ -73,6 +73,20 @@ <h4>
                     to the local variable <code>self</code> remains in scope and is accessible there.
                 </p>
             </li>
+            <li>
```
Comment (arpansac) [inline javascript/index.html:99]: Please shift this into another file, do take a look at the fresh master branch code

**Review by arpansac — CHANGES_REQUESTED** [review]
Please shift the answer into another folder. Refer to master branch. Push to the PR again!

File: javascript/index.html (hunk 1 above)
Comment (Dhanu084) [reply javascript/index.html:99]: Done sir

**Review by Dhanu084 — COMMENTED** [review]
Changes done

**Pull request closed without merging**